docs/setup/custom-config.mdx

```diff
@@ -198,6 +202,35 @@ The `type` parameter specifies the type of caching mechanism to use for storing
 * `local` (default): Stores prediction results in the `cache` folder (as defined in the `paths` configuration). This is useful for repeated queries where the result doesn't change.
 * `redis`: Stores prediction results in a Redis instance. This option requires the `connection` parameter, which specifies the Redis connection string.
 
+The `connection` parameter is required only if the `type` parameter is set to `redis`. It stores the Redis connection string.
+
+```bash
+    "logging": {
+        "handlers": {
+            "console": {
+                "enabled": true,
+                "level": "INFO" # optional (alternatively, it can be defined in the MINDSDB_CONSOLE_LOG_LEVEL environment variable)
+            },
+            "file": {
+                "enabled": False,
+                "level": "INFO", # optional (alternatively, it can be defined in the MINDSDB_FILE_LOG_LEVEL environment variable)
+                "filename": "app.log",
+                "maxBytes": 524288, # 0.5 Mb
+                "backupCount": 3
+            }
+        }
+    },
+```
+
+The above parameters are implemented based on [Python's Dictionary Schema](https://docs.python.org/3/library/logging.config.html#logging-config-dictschema).
```

Comment:
hm, may be better `Python's logging Dictionary Schema`, or `Logging's Dictionary Schema`